entrypoint.R

```diff
@@ -76,7 +76,7 @@ d_dedup <- d_dedup |>
 d_out <- left_join(d_week, d_dedup, by = c("lat", "lon", "week")) |>
   group_by(.row) |>
   summarize(pm = round(sum(pm)/n(),2), 
-            sd = round(sum(sd)/n(),2))
+            sd = round(sqrt((sum(sd^2))/n()),2))
```

Comment:
@cole-brokamp does this calculation look right? Rima's email said:

maybe we need SQRT [ Mean( SUM(SDsquared) )] by week for the SD measure

Reply:
let's review this later this afternoon when we talk, thanks